modules/nf-core/proteinortho/tests/main.nf.test

```diff
@@ -0,0 +1,41 @@
+nextflow_process {
+
+    name "Test Process PROTEINORTHO"
+    script "../main.nf"
+    process "PROTEINORTHO"
+    tag "proteinortho"
+    tag "modules"
+    tag "modules_nfcore"
+
+    test("Should run without failures") {
+
+        when {
+            params {
+                outdir = "$outputDir"
+            }
+            process {
+                """
+                file_a = file(params.test_data['sarscov2']['genome']['proteome_fasta'], checkIfExists: true)
+                file_a.copyTo("${workDir}/prot1.fasta")
+                file_a.copyTo("${workDir}/prot2.fasta")
+                file_a = file("${workDir}/prot1.fasta", checkIfExists:true)
+                file_b = file("${workDir}/prot2.fasta", checkIfExists:true)
+                input[0] = [ 
+                    [ id:'test' ], // meta map
+                    [ 
+                        file_a,
+                        file_b,
+                    ] 
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.orthologgroups).match("orthologgroups") }
```

Comment:
Can you also snapshot the `versions` output channel?

Reply:
I added a assert but without a snapshot, as dependencies could change (diamond) without impacting the results.
And I found a lot of examples with just `{ assert process.out.versions }`
Is this fine?

Reply:
We've discussed this before with the maintainers group and we concluded that snapshotting the versions channel also makes sure that the module snapshots are up to date with their current state

Reply:
ok then I will add this as well with next commit